README.md

```diff
@@ -45,6 +45,8 @@ beta channel to get access to early builds of Desktop:
 
 ## I have a problem with GitHub Desktop
 
+Please read and observe the [GitHub Desktop Code of Conduct](https://github.com/desktop/desktop/blob/master/CODE_OF_CONDUCT.md) in all interactions.
```

Comment:
I would rather we use the phrasing:

> **Note:** The [GitHub Desktop Code of Conduct](https://github.com/desktop/desktop/blob/master/CODE_OF_CONDUCT.md) applies to all interactions with the project team.

I feel it makes it more clear what our intention is.

Reply:
@lee-dohm Thanks! Isn't it also more broad than just the project team though?

Reply:
Good point! Perhaps this instead?

> **Note:** The [GitHub Desktop Code of Conduct](https://github.com/desktop/desktop/blob/master/CODE_OF_CONDUCT.md) applies to all interactions with regard to the GitHub Desktop project.